tests/neo-vm.Tests/Tests/OpCodes/Splice/CAT.json

```diff
@@ -5,19 +5,36 @@
     {
       "name": "Max Item Length",
       "script": [
+        "INITSLOT",
+        "0x0200",
+        "PUSHINT32",
+        "0x00000100",
+        "STLOC0",
         "PUSH1",
-        "PUSHDATA4",
-        "0x00001000",
-        "PUSH0*1048576",
+        "STLOC1",
```

Comment:
@shargon some comments to ease your review
LOC0 = 0x10000 // while loop limit
LOC1 = the counter